Dshop/apps/products_catalogue/urls.py

```diff
@@ -1,10 +1,11 @@
 from django.urls import path
 from django.conf import settings
 from django.conf.urls.static import static
-from .views import ProductListView
+from .views import ProductListView, ProductDetailView
 
 urlpatterns = [
     path('products_list/', ProductListView.as_view(), name='products-list'),
+    path("products/<slug>", ProductDetailView.as_view(), name='product-detail'),
```

Comment:
```suggestion
    path("products/<slug>-<id:int>", ProductDetailView.as_view(), name='product-detail'),
```
Please use slug + id for link. Slugs are not unique.